src/pages/en/guides/cms/buttercms.md

```diff
@@ -0,0 +1,124 @@
+---
+title: ButterCMS & Astro
+description: Add content to your Astro project using ButterCMS
+layout: ~/layouts/CMSLayout.astro
+setup: |
+    import PackageManagerTabs from '~/components/tabs/PackageManagerTabs.astro'
+stub: false
+---
+
+ButterCMS is a headless CMS and blog engine that allows you to publish structured content to use in your project.
+
+## Integrating with Astro
+
+In this section, we'll use the ButterCMS SDK to bring your ButterCMS data into your Astro project.
+
+To get started, you will need to have the following:
+
+### Prerequisites
+
+1. **An Astro project** - If you don't have an Astro project yet, our [Installation guide](/en/install/auto/) will get you up and running in no time.
+
+2. **A ButterCMS account**. If you don't have an account, you can [sign up](https://buttercms.com/join/) for a free trial.
+
+3. **Your ButterCMS API Token** - You can find your API Token on the [Settings](https://buttercms.com/settings/) page.
+
+### Setup
+
+1. Create a `.env` file in the root of your project and add your API token as an environment variable:
+
+```ini title=".env"
+BUTTER_TOKEN=YOUR_API_TOKEN_HERE
+```
+
+:::tip
+Check out [Using environment variables](/en/guides/environment-variables/) for more information about environment variables and `.env` files in Astro.
+:::
+
+2. Install the ButterCMS SDK as a dependency:
+<PackageManagerTabs>
+  <Fragment slot="npm">
+  ```shell
+  npm install buttercms
+  ```
+  </Fragment>
+  <Fragment slot="pnpm">
+  ```shell
+  pnpm install buttercms
+  ```
+  </Fragment>
+  <Fragment slot="yarn">
+  ```shell
+  yarn add buttercms
+  ```
+  </Fragment>
+</PackageManagerTabs>
+
+3. Create a `buttercms.js` file in a new `src/lib/` directory in your project:
+
+```js title="src/lib/buttercms.js"
+import Butter from "buttercms";
+
+export const butterClient = Butter(import.meta.env.BUTTER_TOKEN);
+```
+
+This authenticates the SDK using your API Token and exports it to be used across your project.
+
+### Fetching Data
+To fetch content, import this client and use one of its `retrieve` functions. In this example, we [retrieve a collection](https://buttercms.com/docs/api/#retrieve-a-collection) that has three fields: a short text `name`, a number `price`, and a WYSIWYG `description`. 
```

Comment:
```suggestion
To fetch content, import this client and use one of its `retrieve` functions. 

In this example, we [retrieve a collection](https://buttercms.com/docs/api/#retrieve-a-collection) that has three fields: a short text `name`, a number `price`, and a WYSIWYG `description`. 
```
See how this looks. Wherever it's not too clunky, I like to start the "for example" on its own line, separate from a more general instruction.